applications/nrf5340_audio/dfu/conf/Kconfig.dfu

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config AUDIO_DFU
+	int "Select which DFU type. 0=NONE, 1=Internal flash, 2=External flash"
+	range 0 2
+	default 0
+	help
+	  Setting this variable to 0 disables DFU.
+	  Setting this variable to 1 selects internal flash single image DFU.
+	  Setting to 2 select external flash multi images DFU.
+
+config B0N_MINIMAL
+	bool "B0N use minimal or not"
+	default n
+	help
+	  Let CMakelist choose corresponding overlay file
+
+if AUDIO_DFU = 0
+
+# Avoid multi-build system build network core firmware
+config NCS_INCLUDE_RPMSG_CHILD_IMAGE
+	bool
+	default n
+
+config BT_BUF_ACL_RX_SIZE
```

Comment:
To enable DFU, we need NCS_INCLUDE_RPMSG_CHILD_IMAGE=y
However if this value is set =n in src\bluetooth\ already, it can't be overridden in kconfig.dfu.
So I move NCS_INCLUDE_RPMSG_CHILD_IMAGE to kconfig.dfu, let AUDIO_DFU type to choose enable or not.

Reply:
I don't understand how that relates to the config above: BT_BUF_ACL_RX_SIZE?

Reply:
Remove BT_BUF_ACL_RX_SIZE in Kconfig.dfu.
Add BT_BUF_ACL_RX_SIZE default value if AUDIO_DFU enabled.

> config BT_BUF_ACL_RX_SIZE
> 	int
> 	default 502 if (AUDIO_DFU > 0)
> 	default 259